app/assets/stylesheets/custom.css

```diff
@@ -1,3 +1,11 @@
 h1 {
   text-align: center;
 }
+
+.items {
+  background-color: #d3d3d3;
+  padding: 10px;
+  margin: 20px;
+  border-radius: 25px;
+  text-align: center;
+}
```

Comment:
We're going to have to find a better way to organize our items on the menu. I don't know how, since fuck CSS, but we'll need to spend at least some time going over it.